### ISSUE TITLE
'LinearGaussianCPD' object has no attribute 'variables'

### ISSUE THREAD

**Issue by jainmilan** [pgmpy/pgmpy#890]
I am trying to create a simple linear gaussian model (x1 -> x2) through the following code in v0.1.3:

`model = LinearGaussianBayesianNetwork([('x1', 'x2')])`

`cpd1 = LinearGaussianCPD('x1', [mu_x], sigma_x)`
`cpd2 = LinearGaussianCPD('x2', [beta0, beta_], var_, ['x1'])`

But, when I use following line: 

`model.add_cpds(cpd1, cpd2)`

It says 

> 'LinearGaussianCPD' object has no attribute 'variables'

It seems that there is a bug in the file **LinearGaussianCPD.py**

**Comment by khalibartan** [comment]
Ping @yashu-seth.

**Comment by khalibartan** [comment]
@milanjain81 LinearGaussianBayesianNetwork are not fully supported yet.

**Comment by yashu-seth** [comment]
@khalibartan There is currently a basic model representation support for ```LinearGaussianBayesianNetwork```.

@milanjain81 I don't think it is a bug. There is no ```variables``` attribute in ```LinearGaussianCPD```. There is a ```variable``` attribute, since a linear gaussian CPD is defined on one variable with parents.

**Comment by jainmilan** [comment]
I agree but it seems add_model function is internally calling variables.

**Comment by yashu-seth** [comment]
@milanjain81 Can you please paste your code that is producing the error?

**Comment by jainmilan** [comment]
Hi Yash,

Following code is creating the issue. You may assign random values to mu, sigma, and other variables. 


Best Regards,

Milan Jain
PhD Scholar - Computer Science 
Member: MUC Group <http://muc.iiitd.edu.in/muc/> and Energy Group <http://energy.iiitd.edu.in/>
IIIT Delhi <https://www.iiitd.ac.in/>, Okhla Industrial Estate, Phase III
(Near Govind Puri Metro Station)
New Delhi, India - 110020

Webpage: www.milanjain.com <http://www.milanjain.com/>
LinkedIn: https://www.linkedin.com/in/milanjainr <https://www.linkedin.com/in/milanjainr>

**Comment by khalibartan** [comment]
@yashu-seth Even examples associated with LinearGaussianBayesianNetwork aren't working. Bug is a simple fix since LinearGaussianCPD doesn't have an attribute named variables which is being called by LinearGaussianBayesianNetwork. We are also skipping tests for the same.

**Comment by ankurankan** [comment]
@khalibartan I think I am working on this. Will try to fix this.

**Comment by lohani2280** [comment]
@ankurankan Duplicate issue. Look https://github.com/pgmpy/pgmpy/issues/961 . I think this could be safely closed.

**Comment by theUltraMarine19** [comment]
I agree with @lohani2280
